### PR TITLE
Home page logged in

### PR DESCRIPTION
**Pull request by sarath-yerra**

### CODE REVIEW

**Comment by sarath-yerra** [comment]
Screen Shot 2020-10-28 at 12 08 59 PM

**Comment by sarath-yerra** [comment]
I have commented out the /home feature so its ready for merge into master branch

File: src/Modules/Registration/RegistrationTextInfoComponent.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+import React, { Fragment, useEffect, useState } from 'react';
 import TagManager from 'react-gtm-module'
 
 
 import RegistrationHeaderComponent from './RegistrationHeaderComponent';
 import EventCardComponent from '../Events/EventCardComponent';
+import BackButtonComponent from '../General/BackButtonComponent';
 
 const RegistrationTextInfoComponent = ({event, onRegisterNow}) => {
+const isFBLoggedIn = localStorage.getItem("isFBLoggedIn");
```
Comment (maheshkurma) [inline src/Modules/Registration/RegistrationTextInfoComponent.js:10]: In order to get back the real boolean values, we can use the JSON. parse() method.
here it returns value as "false" but not `false`

Comment (maheshkurma) [reply src/Modules/Registration/RegistrationTextInfoComponent.js]: updated in this commit - https://github.com/midohiofoodbank/freshtrak-client/pull/188/commits/4d07e048e92cee77474cf67b1a71ba4c11e42f04

**Review by maheshkurma — APPROVED** [review]
looks good for now.

**Pull request merged**